test/app.js

```diff
@@ -27,7 +27,7 @@ describe('node:app', function () {
     mockery.enable({warnOnUnregistered: false});
 
     mockery.registerMock('npm-name', function (name, cb) {
-      cb(true);
+      cb(null,true);//no errors, name available
```

Comment:
For sure, had them there while debugging, but certainly not needed now. Will fix those up